homeassistant/components/dsmr_reader/config_flow.py

```diff
@@ -0,0 +1,47 @@
+"""Config flow to configure DSMR Reader."""
+from __future__ import annotations
+
+from collections.abc import Awaitable
+from typing import Any
+
+from homeassistant import config_entries
+from homeassistant.core import HomeAssistant
+
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.config_entry_flow import DiscoveryFlowHandler
+
+from .const import DOMAIN
+
+
+async def _async_has_devices(_: HomeAssistant) -> bool:
+    """Always return true as this integration doesn't support any real devices."""
+    return True
+
+
+@config_entries.HANDLERS.register(DOMAIN)
```

Comment:
```suggestion
```

Untested, but I'm pretty sure you don't need this

Reply:
I'm pretty sure I had to add this in the past, as I got an error saying that no handler was configured for the config flow (at https://github.com/home-assistant/core/blob/dev/homeassistant/config_entries.py#L747). I've tested it again without that line, and it seems the initialization now properly takes care of adding the handler. It'll be removed in the next commit.